src/test/java/org/jitsi/meet/test/pageobjects/mobile/stats/TestConnectionInfo.java

```diff
@@ -47,6 +47,16 @@
     private static final String ID_CONFERENCE_JOINED_STATE
         = "org.jitsi.meet.conference.joinedState";
 
+    private static final String ID_GRANT_MODERATOR_ENABLED
```

Comment:
Let's call this ID_GRANT_MODERATOR_AVAILABLE, since it's not something you can turn off.

Reply:
Updated it, also in the JM PR!